docs/project/get-help.md

```diff
@@ -67,22 +67,27 @@ community members and developers.
 </table>
 
 
-## IRC Quickstart
+# IRC Quickstart 
 
-IRC can also be overwhelming for new users. This quickstart shows you 
-the easiest way to connect to IRC. 
+The following instructions show you how to register with two web based IRC
+tools. Pick one or find another. While these instructions are only for two IRC
+Web Clients there are many IRC Clients available on most platforms.
 
-1. In your browser open <a href="http://webchat.freenode.net" target="_blank">http://webchat.freenode.net</a>
+## Webchat 
 
-    ![Login screen](/project/images/irc_connect.png)
+Using Webchat from Freenode.net is a quick and easy way to get chatting. To
+register:
 
+**Step One** - In your browser open <a href="https://webchat.freenode.net" target="_blank">https://webchat.freenode.net</a>
```

Comment:
Ok... The regular list format was restarting it's numbering from 1 about halfway through, so I searched for any posts on list format control.. That's where I found the instructions to use step 1,2... style.  So I went with that instead of figuring out why the list numbering was starting over.  Will try again.